geps/gep-1713/index.md

```diff
@@ -545,7 +563,195 @@ Listeners should be merged using the following precedence:
 2. ListenerSet ordered by creation time (oldest first)
 3. ListenerSet ordered alphabetically by “{namespace}/{name}”.
 
-Conflicts are covered in the section 'ListenerConditions within a ListenerSet'
+Conflicts are covered in the section [Listener and ListenerSet conflicts](#listener-and-listenerset-conflicts)
+
+### Listener and ListenerSet conflicts
```

Comment:
We should combine this section into `ListenerConditions within a ListenerSet` so that conflicts are covered in one section.

Reply:
IMHO having a different section that defines what a conflict means from "what are the conditions" make it more clear for implementations.

We can probably refer on "ListenerConditions within a ListenerSet" back to the `### Listener and ListenerSet conflicts` to define what a conflict means, but I think that adding all on a single "Conditions" section would overload it and make it confusing.

Reply:
> IMHO having a different section that defines what a conflict means from "what are the conditions" make it more clear for implementations.

Sure sounds good